### DISCUSSION TITLE
WireGuard Script without WGDashboard

### DISCUSSION THREAD

**Discussion by smarthomelawyer** [tteck/Proxmox#263] [Q&A] [answered]
Hi,

First thank you for all of these scripts - I'm just getting started with Proxmox myself and greatly appreciate your efforts. It's made my transition from several Raspberry Pis to a single Thinkcentre m900 nearly seamless. But I was hoping you might still have the WireGuard LXC container script on hand that does not include the wireguard dashboard. (I noticed you added the dashboard to the script back in March, per a fellow user's suggestion.)

 The reason I ask for it without the dashboard is because I'm quite familiar with Pivpn through the CLI and have never used WGdashboard. So I decided to install the current script (with WGdashboard) thinking I could just ignore the user-interface. It was only after I created all of my VPN profiles with split/full tunnels, etc. in the container console that I realized it doesn't work that way. In short, the Pivpn QR code I scanned on the console (pivpn -qr [profile]) does not include the correct configurations, despite the lxc console showing the correct configs.

Hopefully this makes sense, and thank you again.

**Comment by tteck** [comment, marked as answer]
In your fork, remove lines 82 & 91 through 128 in `/setup/wireguard-install.sh`

**Reply by tteck** [reply]
Change line 346 in `/ct/wireguard-v3.sh` to your github repository

**Reply by tteck** [reply]
Then, install by running
```
bash -c "$(wget -qLO - https://github.com/smarthomelawyer/Proxmox/raw/main/ct/wireguard-v3.sh)"
```

**Reply by smarthomelawyer** [reply]
You sir, are a saint! Thank you for very much! I really appreciate it
